V0_PCB_Mod/PCBs/readme.md

```diff
@@ -0,0 +1,15 @@
+Be careful and take your time to assemble the boards.
+
+The circuit boards were developed, built and tested in my own Voron0 printer by me.
+
+I do not assume any guarantee or liability for mistakes made by others.
+
+The replica is done at your own risk.
+
+The circuit boards are for private use only
+
+All parts can be obtained from Mouser or Digikey.
+
+![Image 1](Fan_PCB/Images/1.jpg)
+![Image 2](Hotend_PCB/Images/1.jpg)
+![Image 3](Supply_PCB/images/1.jpg)
```

Comment:
I think this should be Images with an. upper-case I. The image fails to load in the preview.

Reply:
Thanks for the hint. I will correct the mistake.